src/templates/article.tsx

```diff
@@ -1,49 +1,79 @@
-import React from "react"
-import { graphql } from "gatsby"
-import { RichText } from "prismic-reactjs"
+import React, { ReactElement } from "react"
 
 import Layout from "../components/layout"
 import SEO from "../components/seo"
 import Author from "../components/author"
 import CustomRichText from "../components/customRichText"
 import { formatHumanDate } from "../utils/date"
+import { RichText, RichTextBlock } from "prismic-reactjs"
 
-export default function Article(props) {
-  const doc = props.data.prismic.allBlog_posts.edges.slice(0, 1).pop()
-
-  if (!doc) return null
+interface PropsArticle {
+  data: {
+    title: {
+      raw: RichTextBlock[]
+    }
+    description: {
+      raw: RichTextBlock[]
+    }
+    content: {
+      text: string
+      raw: RichTextBlock[]
+      html: string
+    }
+    date: string
+    image: {
+      url: string
+      alt: string
+      copyright: string
+      gatsbyImageData
+    }
+  }
+}
 
-  const title = RichText.asText(doc.node.title)
-  const description = RichText.asText(doc.node.description)
-  const datePublished = formatHumanDate(doc.node.date)
+export default function Article(props: PropsArticle): ReactElement {
+  // if (!doc) return null
+  const datePublished = formatHumanDate(props.pageContext.data.date)
 
-  const { image } = doc.node
+  const image = props.pageContext.data.image.url
```

Comment:
Like in `articleListItem.tsx` it's a best practice to create new variables like

```
const title = props.pageContext.data.title.text
const description = props.pageContext.data.description.text
```

later we can use
```
      <SEO
        title={title}
        description={description}
      />
```
Because if we need to change the API some day we will need to update only once instead of multiple times